diff --git a/node_modules/resolve-url-loader/lib/engine/postcss.js b/node_modules/resolve-url-loader/lib/engine/postcss.js
index 87571f1..601d2bc 100644
--- a/node_modules/resolve-url-loader/lib/engine/postcss.js
+++ b/node_modules/resolve-url-loader/lib/engine/postcss.js
@@ -30,7 +30,7 @@ function process(sourceFile, sourceContent, params) {
 
   // prepend file protocol to all sources to avoid problems with source map
   return postcss([
-    postcss.plugin('postcss-resolve-url', postcssPlugin)
+    postcssPlugin()
   ])
     .process(correctedContent, {
       from: fileProtocol.prepend(sourceFile),
